Description: Fix paths used by upstream
 Logs should go into '/var/log/schleuder'.
 
Author: Georg Faerber <georg@riseup.net>
Reviewed-by: Sebastien Badia <sbadia@debian.org>
Last-Update: 2017-01-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/schleuder.yml
+++ b/etc/schleuder.yml
@@ -2,7 +2,7 @@
 lists_dir: /var/lib/schleuder/lists
 
 # Where to write list-logs. The actual log-file will be <lists_logs_base_dir>/<hostname>/<listname>/list.log.
-listlogs_dir: /var/lib/schleuder/lists
+listlogs_dir: /var/log/schleuder/lists
 
 # Schleuder reads plugins also from this directory.
 plugins_dir: /etc/schleuder/plugins
